modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/PagedOperationExecutionTestCase.java

```diff
@@ -121,7 +121,7 @@ public void pagingProviderIsClosedSafelyDuringExceptionOnFirstPage() throws Exce
   @Test
   public void pagingProviderIsClosedSafelyAfterDataSourceIsFullyConsumed() throws Exception {
     resetCounters();
-    flowRunner("consumeFailAtClosePagedOperation").withPayload(4).run();
+    flowRunner("consumeFailAtClosePagedOperation").withPayload(-1).run();
```

Comment:
why this change? Weird that no other assertion changed

Reply:
The 4 is a parameter indicating that on the 4th page it would fail, but on the 3rd page everything is consumed and there are no more pages, thus it does not fail. I think the -1 is more explicit of the intention "to never fail". I can revert it tho